File: ObjectProgrammingJava/projekt po - 2/src/projekt/obsluga/UstawiaczOrganizmow.java
package projekt.obsluga;

import projekt.Organizm;
import projekt.rosliny.*;
import projekt.swiat.Swiat;
import projekt.zwierzeta.*;

public class UstawiaczOrganizmow {

    public Organizm klonowanie(String nazwa, Swiat swiat, int x, int y){
        if(nazwa.equals("Wilk")) {
            return new Wilk(swiat, x, y, true);
        }
        if(nazwa.equals("Owca")) {
            return new Owca(swiat, x, y, true);
        }
        if(nazwa.equals("Lis")) {
            return new Lis(swiat, x, y, true);
        }
        if(nazwa.equals("Antylopa")) {
            return new Antylopa(swiat, x, y, true);
        }
        if(nazwa.equals("Zolw")) {
            return new Zolw(swiat, x, y, true);
        }
        if(nazwa.equals("Trawa")) {
            return new Trawa(swiat, x, y, true);
        }
        if(nazwa.equals("Mlecz")) {
            return new Mlecz(swiat, x, y, true);
        }
        if(nazwa.equals("Jagody")) {
            return new Jagody(swiat, x, y, true);
        }
        if(nazwa.equals("Barszcz")) {
            return new Barszcz(swiat, x, y, true);
        }
        if(nazwa.equals("Guarana")) {
            return new Guarana(swiat, x, y, true);
        }
        return null;
    }
}

File: ObjectProgrammingPython/Obsluga/ustawiaczOrganizmow.py
import Zwierzeta
import Rosliny


class UstawiaczOrganizmow:

    def klonowanie(self, nazwa, swiat, x, y):
        if nazwa == "Wilk":
            return Zwierzeta.wilk.Wilk(swiat, x, y, True)
        if nazwa == "Owca":
            return Zwierzeta.owca.Owca(swiat, x, y, True)
        if nazwa == "Antylopa":
            return Zwierzeta.antylopa.Antylopa(swiat, x, y, True)
        if nazwa == "Lis":
            return Zwierzeta.lis.Lis(swiat, x, y, True)
        if nazwa == "Zolw":
            return Zwierzeta.zolw.Zolw(swiat, x, y, True)
        if nazwa == "Trawa":
            return Rosliny.trawa.Trawa(swiat, x, y, True)
        if nazwa == "Mlecz":
            return Rosliny.mlecz.Mlecz(swiat, x, y, True)
        if nazwa == "Guarana":
            return Rosliny.guarana.Guarana(swiat, x, y, True)
        if nazwa == "Jagody":
            return Rosliny.jagody.Jagody(swiat, x, y, True)
        if nazwa == "Barszcz":
            return Rosliny.barszcz.Barszcz(swiat, x, y, True)
        if nazwa == "Cyber owca":
            return Zwierzeta.cyberOwca.CyberOwca(swiat, x, y, True)
